### PR TITLE
Rename file extension of RunnerTemplate.Windows.txt to .cmd

### PR DESCRIPTION
**Pull request by akoeplinger**
So it automatically gets the EOL treatment from #1336.

---

Original description:

After https://github.com/dotnet/runtime/pull/1253 was merged I got the following error when running build-native.cmd on my machine:

```
C:\Users\Alexander\Dev\runtime> src\libraries\Native\build-native.cmd x64 Debug Windows_NT outconfig netcoreapp5.0-Windows_NT-Debug-x64
**********************************************************************
** Visual Studio 2019 Developer Command Prompt v16.5.0-pre.2.0
** Copyright (c) 2019 Microsoft Corporation
**********************************************************************
[vcvarsall.bat] Environment initialized for: 'x86_x64'
Commencing build of native components

The system cannot find the batch label specified - GenVSSolution
```

Adding a comment before the exit call fixes this though I'm not fully sure why. Maybe the CMD parser got confused.

### CODE REVIEW

**Comment by akoeplinger** [comment]
Hmm I'm seeing this in more places, e.g. RunTests.cmd too. Sounds like my machine might have an issue or has anyone else seen this?

**Comment by jkotas** [comment]
I have not seen it, but I know that .cmd files tend to break in unpredictable ways like this one.

@danmosemsft Sounds like there is something wrong with #1253 ...

**Comment by tannergooding** [comment]
This may be the same reason as why I did https://github.com/dotnet/runtime/pull/1336 and modifying the file just added the needed crlf further label to be correctly found.

Recloning the repo to see if that resolves the issue may be a good idea.

**Comment by akoeplinger** [comment]
@tannergooding good catch, that was indeed the issue! I found that the RunnerTemplate.Windows.txt was still LF though since it doesn't have .cmd extension, I fixed that in this PR.

**Review by jkotas — APPROVED** [review]
Thanks!

File: eng/testing/tests.targets [hunk 1]
```diff
@@ -3,8 +3,8 @@
     <TargetOS Condition="'$(TargetOS)' == ''">$(DefaultOSGroup)</TargetOS>
     <RunWorkingDirectory>$(OutDir)</RunWorkingDirectory>
 
-    <RunScriptInputName Condition="'$(TargetOS)' == 'Windows_NT'">RunnerTemplate.Windows.txt</RunScriptInputName>
-    <RunScriptInputName Condition="'$(TargetOS)' != 'Windows_NT'">RunnerTemplate.Unix.txt</RunScriptInputName>
+    <RunScriptInputName Condition="'$(TargetOS)' == 'Windows_NT'">RunnerTemplate.Windows.cmd</RunScriptInputName>
```
Comment (jkotas) [inline eng/testing/tests.targets]: Nit: We do not need the "Windows" and "Unix" in the name anymore

**Pull request merged**